Description: Re-enable ensurepip inside this package
Author: Sebastian Muszynski <basti@linkt.de>

--- a/debian/rules
+++ b/debian/rules
@@ -334,7 +334,7 @@
 	test_threading test_threading_local test_threadsignals \
 	test_concurrent_futures test_ctypes \
 	test_dbm_dumb test_dbm_ndbm test_pydoc test_sundry \
-	test_signal test_ioctl test_gdb test_ensurepip test_venv
+	test_signal test_ioctl test_gdb test_venv
 
 # TODO: these hang/fail with new OpenSSL
 PROFILE_EXCLUDES += \
@@ -583,7 +583,7 @@
 endif
 TESTOPTS = -j 1 -w -u$(TEST_RESOURCES) --timeout=3600
 TEST_EXCLUDES =
-TEST_EXCLUDES += test_ensurepip test_venv
+TEST_EXCLUDES += test_venv
 ifeq ($(on_buildd),yes)
   TEST_EXCLUDES += test_tcl test_codecmaps_cn test_codecmaps_hk \
         test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw \
@@ -920,10 +920,6 @@
 	rm -rf $(d)/usr/lib/python$(VER)/ctypes/macholib
 	rm -f $(d)/usr/share/man/man1/python3.1
 
-	if [ -d $(d)/$(scriptdir)/ensurepip/_bundled ]; then \
-	  echo "don't ship the prebuilt ensurepip/_bundled wheels."; \
-	  rm -rf $(d)/$(scriptdir)/ensurepip/_bundled; \
-	fi
 
 	: # cannot build it, zlib maintainer won't provide a mingw build
 	find $(d) -name 'wininst*.exe' | xargs -r rm -f
--- a/Lib/ensurepip/__init__.py
+++ b/Lib/ensurepip/__init__.py
@@ -16,7 +16,7 @@
 # policies recommend against bundling dependencies. For example, Fedora
 # installs wheel packages in the /usr/share/python-wheels/ directory and don't
 # install the ensurepip._bundled package.
-if (_pkg_dir := sysconfig.get_config_var('WHEEL_PKG_DIR')) is not None:
+if (_pkg_dir := sysconfig.get_config_var('DO_NOT_USE_WHEEL_PKG_DIR')) is not None:
     _WHEEL_PKG_DIR = Path(_pkg_dir).resolve()
 else:
     _WHEEL_PKG_DIR = None
--- a/debian/control
+++ b/debian/control
@@ -59,8 +59,6 @@
 Multi-Arch: allowed
 Priority: optional
 Depends: python3.14 (= ${binary:Version}),
- python3-pip-whl (>= 22.2),
- python3-setuptools-whl (>= 70.1),
  ${shlibs:Depends}, ${misc:Depends},
 Breaks: python3-pip (<< 1.5.6-4)
 Description: Interactive high-level object-oriented language (pyvenv binary, version 3.14)
--- a/debian/control.in
+++ b/debian/control.in
@@ -59,8 +59,6 @@
 Multi-Arch: allowed
 Priority: @PRIO@
 Depends: @PVER@ (= ${binary:Version}),
- python3-pip-whl (>= 22.2),
- python3-setuptools-whl (>= 70.1),
  ${shlibs:Depends}, ${misc:Depends},
 Breaks: python3-pip (<< 1.5.6-4)
 Description: Interactive high-level object-oriented language (pyvenv binary, version @VER@)
